packages/fluentui/react-northstar/src/components/Carousel/Carousel.tsx

```diff
@@ -364,6 +364,7 @@ export const Carousel: ComponentWithAs<'div', CarouselProps> &
   };
 
   const handlePaddleOverrides = (predefinedProps: CarouselPaddleProps, paddleName: string) => ({
+    variables,
```

Comment:
Variables should be merged 

```suggestion
    variables: mergeVariablesOverrides(variables, predefinedProps.variables),
```